x/tally/types/params.go

```diff
@@ -9,6 +9,7 @@ const (
 	DefaultFilterGasCostNone             = 100_000
 	DefaultFilterGasCostMultiplierMode   = 100_000
 	DefaultFilterGasCostMultiplierStddev = 100_000
+	DefaultGasCostCommitment             = 100_000_000
```

Comment:
This should probably be the same as the DR VM startup cost which is  `5_000_000_000_000` right now, but subject to change as we tweak the gas values :)

Reply:
fixed, thanks!!